notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb

```diff
@@ -32,18 +32,18 @@
         "<table align=\"left\">\n",
```

Comment:
<p>Is this specific to BQML models?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/993/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>Not directly. It's for users to confirm their model is uploaded to Vertex Model Registry.</p>

Reply:
ok, I will discuss offline how this differs from list(filter=....) or loading using the full model ID.

Reply:
Sounds good :)